boltdesignsystem/bolt
# Hotfix/DS-238 Update v3 dependencies

## Jira

https://pegadigitalit.atlassian.net/browse/DS-238

## Summary

A few dependency updates for the `3.x` release.

## Details

- Remove `build-tools` dependency from Microjourneys (see original PR #1993). This should not be required. Will be tested on .COM.
- Upgrade `@bolt/testing-utils` which was still on `v2.x`.
- Remove PL's `uikit-polyfills` and use our own `@bolt/polyfills`.

## How to test

- Build is passing.
- Check out feature branch locally and verify custom-elements still work in Edge.
